decidim-core/app/controllers/decidim/scopes_controller.rb

```diff
@@ -13,7 +13,7 @@ def search
                else
                  current_organization.top_scopes
                end
-      root_option = if params[:include_root] && !params[:term].present?
+      root_option = if params[:include_root] && params[:include_root] != "false" && !params[:term].present?
```

Comment:
Actually, one more thing regarding the `include_root` parameter. Since as you said, we're never passing `include_root: false`, I think it's clearer (and simpler) to explicitly check for the positive case. Would this condition be equivalent?

```diff
                else
                  current_organization.top_scopes
                end
-      root_option = if params[:include_root] && params[:include_root] != "false" && !params[:term].present?
+      root_option = if params[:include_root] == "true" && !params[:term].present?
                       if root
                         [{ id: root.id.to_s, text: root.name[I18n.locale.to_s] }]
                       else
```